doc/changelog/10-standard-library/11946-ollibs-permutation.rst

```diff
@@ -0,0 +1,10 @@
+- **Added:**
+  Facts about ``Permutation``:
+
+  - structure: ``Permutation_refl'``, ``Permutation_morph_transp``
+  - compatibilities: ``Permutation_app_rot``, ``Permutation_app_swap_app``, ``Permutation_app_middle``, ``Permutation_middle2``, ``Permutation_elt``, ``Permutation_Forall``, ``Permutation_Exists``, ``Permutation_Forall2``, ``Permutation_flat_map``, ``Permutation_list_sum``, ``Permutation_list_max``
+  - inversions: ``Permutation_app_inv_m``, ``Permutation_vs_elt_inv``, ``Permutation_vs_cons_inv``, ``Permutation_vs_cons_cons_inv``, ``Permutation_map_inv``, ``Permutation_image``, ``Permutation_elt_map_inv``
+  - length-preserving definition by means of transpositions: ``Permutation_transp``
```

Comment:
Thank you! Should we also add `Permutation_transp_sym`, `Permutation_transp_equiv`, `Permutation_transp_cons`, `Permutation_Permutation_transp`, `Permutation_ind_transp`?

Reply:
OK I will.
I was not sure whether it should be exhaustive or more compact for readability (the missing names correspond to properties of the new `Permutation_transp` thus I mentioned it alone).